packages/x-date-pickers/src/internals/components/FakeTextField/fakeTextFieldClasses.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  unstable_generateUtilityClass as generateUtilityClass,
+  unstable_generateUtilityClasses as generateUtilityClasses,
+} from '@mui/utils';
+
+export interface FakeTextFieldClasses {
+  /** Styles applied to the root element. */
+  root: string;
+  /** Styles applied to the root element if focused. */
+  focused: string;
+  /** State class applied to the root element if `disabled=true`. */
+  disabled: string;
+  /** State class applied to the root element if `error=true`. */
+  error: string;
+  /** State class applied to the root element id `required=true` */
+  required: string;
+}
+export type FakeTextFieldClassKey = keyof FakeTextFieldClasses;
+
+export function getFakeTextFieldUtilityClass(slot: string) {
+  return generateUtilityClass('MuiFakeTextField', slot);
+}
+
+export const fakeTextFieldClasses = generateUtilityClasses<FakeTextFieldClassKey>(
+  'MuiFakeTextField',
```

Comment:
@LukasTy @noraleonte how do you think we should call this component (and its children)
I named it `Fake` but it's not really a production name.

Reply:
`MultiSectionTextField` & `MultiSectionInput`? 
I agree that `Fake` is a weird naming. The name should suggest what the component does and why it's different from the material ui `TextField`. I like the idea of `Multi[something]Field` or `Multi[something]Input]` 🤔

Reply:
`AccessibleTemporalTextField`? :laughing: